src/angularComponent/ngcOmniboxController.js

```diff
@@ -200,6 +200,19 @@ export default class NgcOmniboxController {
     }
   }
 
+  /**
+   * Removes a particular suggestion item frmo the list of selected items.
```

Comment:
I mean, when multiple is off it's still a list, just a list of one, but it's fine, I don't care enough to fight you on it.